### ISSUE TITLE
14 crashes after restoring from sleep mode in XP. It did not happen in 13.

### ISSUE THREAD

**Issue by etsenberg** [Feodor2/Mypal68#452]
I hoped 14b would fix this 14,0 issue, but it didn't, In happens not after every restore but about every 4 or 5 restore.  The most common error reported in application events is either 
error ID 1000, mypal.exe, version 68.12.0.8890, module xul.dll, version 68.12.0.8890, address 0x00af2dc7.
or 
error ID 1000, mypal.exe, version 68.12.0.8890, module d3d9.dll, version 5.3.2600.5512, address 0x000858c3.

I don't know why it says version 68.12.0.8890 while I run 68.14.1b.

**Comment by RamonUnch** [comment]
I think this is simply related to hardware acceleration, in previous release d3d9 was probably not used with your config, try disabling it as explained in the release page:
https://github.com/Feodor2/Mypal68/releases/tag/68.14.0b

**Comment by Feodor2** [comment]
Yes, disable hardware acceleration. 
Also please tell you videocard model and give the drwatson.log

**Comment by etsenberg** [comment]
No, my graphic settings are the same as in 13 because it's the same profile. And I don't want to make my slow computer even slower in graphic pages by turning hardware acceleration off. I can do for test purpose (the result will be clear in several days), but it cannot be a permanent solution, 
My card is NVIDIA GeForce GT 730
drwtsn32.log is a 21M file with lots of entries mentioning mypal, which parts of it do you need? I tried to extract the part related to the last crash, but I am not sure if I did it correctly. Sorry for non-English words in the log. In the very beginning, it says "exception c0000094 (division by zero)"
[drwatson.txt](https://github.com/Feodor2/Mypal68/files/15359117/drwatson.txt)

I also attached my dxdiag info, may it help.
[DxDiag.txt](https://github.com/Feodor2/Mypal68/files/15359107/DxDiag.txt)

**Comment by RamonUnch** [comment]
You can check in `about:support` if hardware composition was enabled with the default config of older versions, but I think that the`layers.allow-d3d9-fallback` was off by default before.

**Comment by etsenberg** [comment]
> You can check in `about:support` if hardware composition was enabled with the default config of older versions, but I think that the`layers.allow-d3d9-fallback` was off by default before.

I don't see any parameter named layers.allow-d3d9-fallback there. I see only 
layers.acceleration.disabled	true (no wonder after I disabled it in about:config)
and
layers.mlgpu.sanity-test-failed	false

**Comment by RamonUnch** [comment]
`layers.allow-d3d9-fallback` is an `about:config` option I was proposing you checked the old version of MyPal with a clean profile and then check in `about:support` to see if the hardware acceleration was enabled.

**Comment by etsenberg** [comment]
> `layers.allow-d3d9-fallback` is an `about:config` option I was proposing you checked the old version of MyPal with a clean profile and then check in `about:support` to see if the hardware acceleration was enabled.

Indeed, false in 13.7, true in 14.

**Comment by zanud** [comment]
On my notebook (with Intel integrayed video) Mypal 68.14.1 crashed after restoring from hibernation (I do not use sleep mode).

The culprit was `layers.acceleration.force-enabled = true`.
